tests/src/test/kotlin/com/android/declarative/tests/DeclarativeTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.android.declarative.tests
+
+import com.google.common.truth.Truth
+import java.io.File
+import java.nio.file.Files
+
+/**
+ * Adds a gradle wrapper declaration, using the one located under
+ * TEST_ROOTDIR environment variable.
+ */
+fun addGradleWrapper(testRootFolder: File) {
```

Comment:
Should the file name be like GradleWrapperUtil instead of DeclarativeTest?

Reply:
so far, that's all it does but I think there will more services added there.